README.md

```diff
@@ -1,404 +1,5 @@
-
-```
--e NODE_EXTRA_CA_CERTS=/home/node/sourcegraph.example.com.crt -v ~/.sourcegraph/config:/home/node
-```
-
-The self signed certificate's `Common Name (CN)` should be the host name of your host. Also make sure you use Websocket with SSL in your Sourcegraph settings to connect to the language server:
-
-```json
-"typescript.serverUrl": "wss://localhost:8080"
-```
-
-### Authentication proxies and firewalls
-
-Some customers deploy Sourcegraph behind an authentication proxy or firewall. If you do this, we
-recommend deploying the language server behind the proxy so that it can issue requests directly to
-Sourcegraph without going through the proxy. (Otherwise, you will need to configure the language
-server to authenticate through your proxy.) Make sure you set `typescript.sourcegraphUrl` to the URL
-that the language server should use to reach Sourcegraph, which is likely different from the URL
-that end users use.
-
-### Using Kubernetes
-
-If you want to deploy the language server with Kubernetes, your deployment should look like this:
-
-```yaml
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: lang-typescript
-spec:
-  replicas: 4 # adjust as needed
-  selector:
-    matchLabels:
-      app: lang-typescript
-  template:
-    metadata:
-      labels:
-        app: lang-typescript
-    spec:
-      containers:
-        - name: lang-typescript
-          image: sourcegraph/lang-typescript
-          ports:
-            - containerPort: 8080
-              name: wss
-          env:
-            # TLS certificate and key to secure the WebSocket connection (optional)
-            - name: TLS_CERT
-              value: ... your TLS certificate ...
-            - name: TLS_KEY
-              value: ... your TLS key ...
-          # Resources to provision for the server (adjust as needed)
-          resources:
-            limits:
-              cpu: '4'
-              memory: 5Gi
-            requests:
-              cpu: 500m
-              memory: 2Gi
-          # Probes the server periodically to see if it is healthy
-          livenessProbe:
-            initialDelaySeconds: 30
-            tcpSocket:
-              port: wss
-            timeoutSeconds: 5
-          readinessProbe:
-            tcpSocket:
-              port: wss
-```
-
-With a corresponding service:
-
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  labels:
-    app: lang-typescript
-    deploy: lang-typescript
-  name: lang-typescript
-spec:
-  ports:
-    - name: wss
-      port: 443
-      targetPort: wss
-  selector:
-    app: lang-typescript
-  type: LoadBalancer
-```
-
-#### TLS
-
-TLS is optional but recommended for production deployments. It is used if `TLS_KEY` and `TLS_CERT` environment variables are set.
-
-#### Enabling OpenTracing
-
-The server can report spans through OpenTracing to diagnose issues.
-If the environment variable `LIGHTSTEP_ACCESS_TOKEN` is set, the server will send tracing data to the given LightStep instance.
```

Comment:
I just noticed that this PR removed all the documentation - while it was all written considering the TS support as one unit, a lot of it was specifically about the server, e.g. that it supports OpenTracing and Prometheus and how to configure it, which settings it accepts,  how to contribute, ...

Could those be brought back? If it's too much work I think it would still be better to have the previous version of the doc (with a note and link at the top that this is the server component of the extension) than to not have any. @sourcegraph/code-intel do you agree?